js/networking/user.js

```diff
@@ -0,0 +1,23 @@
+import { getAuthJson, patchAuthJson } from './networking';
+import { createJsonDataAttributes } from './json';
+/*
+Class with methods for manipulating user info against API.
+*/
+
+const BASE_URL = 'https://sandbox-api.justarrived.xyz';
```

Comment:
Saw this URL in `Auth.js` perhaps these constants should be unified somewhere? :)